server.js

```diff
@@ -1,6 +1,18 @@
 const express = require('express');
 const app = express();
 const port = 3000; 
+const mongoose = require('mongoose');
+const URI = 'mongodb+srv://somuyak:%40Somuya2004@cluster0.i2b8prd.mongodb.net/?retryWrites=true&w=majority'
```

Comment:
Move the database connection string to an environment variable. This will help in maintaining security and flexibility of the application. [important]